components/tor/onion_location_navigation_throttle.cc

```diff
@@ -86,7 +89,8 @@ OnionLocationNavigationThrottle::WillProcessResponse() {
     }
     // If user prefers opening it automatically
     if (pref_service_->GetBoolean(prefs::kAutoOnionRedirect)) {
-      delegate_->OpenInTorWindow(navigation_handle()->GetWebContents(), url);
+      delegate_->OpenInTorWindow(navigation_handle()->GetWebContents(), url,
+                                 false);
```

Comment:
@fmarier why do we keep loading the page via https here? Spec says:
```
a) If the user has enabled automatic Onion-Location redirects
      the header is equivalent to a redirect with a Refresh header and a
      timeout of 0 seconds [1]. As an example: the header in 2.1 would be
      treated like a `Refresh: 0;URL='http://vwc43ag5jyewlfgf.onion'` header.
```

which means the https loading should stop. In our case we cannot reuse the same tab for `.onion` URL by adding `Refresh` header, but I think we should stop the loading here.

I think we can technically do a redirect via throttler(?) which should just close the current tab and open Tor window with the `.onion` URL.

Reply:
I must missed that when implementing it.
Will `content::NavigationThrottle::BLOCK_RESPONSE` suffice?

Reply:
Yes, it will, I can easily add this change.